docs/static/monitoring/monitoring-ea-dashboards.asciidoc

```diff
@@ -0,0 +1,109 @@
+[[dashboard-monitoring-with-elastic-agent]]
+=== Collect {ls} monitoring data for dashboards
+[subs="attributes"]
+++++
+<titleabbrev>Collect monitoring data for dashboards</titleabbrev>
+++++
+
+ToDo: Add brief intro describing the feature and value prop. 
+So far, we haven't differentiated between the two. 
+Add info to help them decide which one they need.  
+
+**Prerequisites**
+
+Complete these steps as you prepare to collect and ship monitoring data for dashboards:
+
+[[disable-default-db]]
+.Disable default collection of {ls} monitoring metrics
+[%collapsible]
+====
+include::monitoring-prereq-disable-default.asciidoc[]
+====
+
+[[define-cluster__uuid-db]]
+.Specify the target cluster_uuid (optional)
+[%collapsible]
+====
+include::monitoring-prereq-define-cluster.asciidoc[]
+====
+
+
+[discrete]
+[[install-and-configure-db]]
+=== Install and configure {agent}
+
+Install and configure {agent} to collect {ls} monitoring data for dashboards.
+We'll walk you through the process in these steps:
+
+* <<add-logstash-integration-ead>>
+* <<add-agent-to-fleet-ead>>
+* <<view-assets-ead>>
+* <<view-data-dashboard>>
+
+[discrete]
+[[add-logstash-integration-ead]]
+==== Add the {agent} {ls} integration to monitor host logs and metrics
+
+. Go to the {kib} home page, and click **Add integrations**.
++
+--
+[role="screenshot"]
+image::images/kibana-home.png[{kib} home page]
+--
```

Comment:
What do you think about having a widget in a follow-up PR that shows the serverless flow?

Reply:
I agree with the "follow-up PR" idea, but I have a different approach I'll run by you.